docs/content/templates/functions.md

```diff
@@ -373,6 +373,15 @@ e.g.
        {{ .Content }}
     {{ end }}
 
+
+### uniq
+
+Loops through an array and creates a new array with only the first occurrence of any given value.
```

Comment:
Generally, try to make the docs description match the func's godoc comment.  We eventually plan to auto-generate these from godoc.